adapters.go

```diff
@@ -229,14 +233,26 @@ func (a *whisperAdapter) SendPublic(ctx context.Context, chatName, chatID string
 		return nil, errors.Wrap(err, "failed to encode message")
 	}
 
+	if a.featureFlags.sendV1Messages {
```

Comment:
This is in two places already. Alternatively, we could add `a.encodeMessage(message)` method which would do the wrapping if the flag is set. 

But this is fine as well.

Reply:
good idea